src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -201,6 +204,46 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
     }
 
+    private mountExpressionTab(){
```

Comment:
out of curiosity, why are you doing it this way rather than the existing idiom we have for mounting react tabs?

Reply:
It's temporary.  Allows me to integrate without making/deploying changes to backend prematurely.